test/test_jujulog.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+
+import unittest
+import importlib
+
+import logging
+import ops.jujulog
+
+
+class FakeModelBackend:
+
+    def __init__(self):
+        self._calls = []
+
+    def calls(self, clear=False):
+        calls = self._calls
+        if clear:
+            self._calls = []
+        return calls
+
+    def juju_log(self, message, level):
+        self._calls.append((message, level))
```

Comment:
Thinking through this, I find it odd that the level comes after the message. A more common paradigm would be:
juju_log(self, level, message, **kwargs):

wouldn't it?
I realize it doesn't really matter for when we are already using the python framework to turn the log messages into strings, but it still feels like 'level' should be the first parameter.

Reply:
I had an idea of having a default level in mind and placed the level parameter at the end but then dropped it in favor of being explicit because it didn't seem like a good idea to support "just logging something with some default level".

If the `level` parameter goes first we pretty much carve it in stone. I can just change the order in a follow-up commit.

Reply:
Addressed in https://github.com/canonical/operator/pull/133/commits/835fa5805e44699a8a1057bc40b4b3cfb99b8f1f